baremetal/metal3data_manager_test.go

```diff
@@ -1258,16 +1307,16 @@ var _ = Describe("Metal3Data manager", func() {
 			poolRef:         corev1.TypedLocalObjectReference{Name: testPoolName},
 			expectedAddress: addressFromPool{},
 			ipClaim: &ipamv1.IPClaim{
-				ObjectMeta: testObjectMeta("abc-abc", namespaceName, ""),
+				ObjectMeta: testObjectMeta(metal3DataName+"-"+testPoolName, namespaceName, ""),
 				Status: ipamv1.IPClaimStatus{
 					Address: &corev1.ObjectReference{
 						Name:      "abc-192.168.0.11",
 						Namespace: namespaceName,
 					},
 				},
 			},
-			expectRequeue: false,
-			expectError:   true,
+			expectRequeue: true,
+			expectError:   false,
```

Comment:
Please double check! This was changed in https://github.com/metal3-io/cluster-api-provider-metal3/commit/42713219f82f33ca625eb7489a210709a000c162# and I think it was a mistake.
The test is supposed to check "IP address not found". My understanding is that this should result in requeue, not error, as seen [here](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/baremetal/metal3data_manager.go#L566).
The reason we got an error is because the name was wrong above so it didn't find the *M3DT*, which would return an error.
/cc @furkatgofurov7

Reply:
Yeah, should be fine now